src/main/resources/templates/db_scripts/populateRetainLeadingZeroesSetting.sql

```diff
@@ -4,6 +4,8 @@ UPDATE ${myuniversity}_${mymodule}.hrid_settings
   SET jsonb = (SELECT jsonb_insert(jsonb, '{commonRetainLeadingZeroes}','true', true)
                FROM ${myuniversity}_${mymodule}.hrid_settings
                WHERE id = 'a501f2a8-5b31-48b2-874d-2191e48db8cd')
-WHERE id = 'a501f2a8-5b31-48b2-874d-2191e48db8cd';
+WHERE id = 'a501f2a8-5b31-48b2-874d-2191e48db8cd'
+     AND (SELECT jsonb::json->'commonRetainLeadingZeroes'
+      FROM ${myuniversity}_${mymodule}.hrid_settings WHERE id = 'a501f2a8-5b31-48b2-874d-2191e48db8cd') IS NULL;
```

Comment:
```suggestion
WHERE id = 'a501f2a8-5b31-48b2-874d-2191e48db8cd' and jsonb->>'commonRetainLeadingZeroes' is NULL;
```

OR:
```suggestion
UPDATE ${myuniversity}_${mymodule}.hrid_settings
  SET jsonb = jsonb_set(jsonb, '{commonRetainLeadingZeroes}', TO_JSONB(true))
WHERE id = 'a501f2a8-5b31-48b2-874d-2191e48db8cd;	
```

Reply:
@bohdan-suprun , thank you. Your decision is so much easier.